src/database/schema/UserSchema.ts

```diff
@@ -140,7 +147,8 @@
         anon_user_info_emoji_name = ?,
         anon_user_info_emoji_code = ?,
         anon_user_info_color_name = ?,
-        anon_user_info_color_code = ?
+        anon_user_info_color_code = ?,
+        is_anonymous = ?
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, please ensure that you are handling null values for `is_anonymous` during update operations as well.

```diff
-        anon_user_info_color_code = ?,
-        is_anonymous = ?
+        anon_user_info_color_code = ?,
+        is_anonymous = ?
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->